projects/angular/src/lib/maskito.cva.ts

```diff
@@ -20,8 +20,16 @@ import {MASKITO_DEFAULT_OPTIONS, MaskitoOptions, maskitoTransform} from '@maskit
     },
 })
 export class MaskitoCva {
+    private options?: MaskitoOptions | null;
+
     @Input()
-    maskito: MaskitoOptions = MASKITO_DEFAULT_OPTIONS;
+    set maskito(options: MaskitoOptions | null | undefined) {
+        this.options = options;
+    }
+
+    get maskito(): MaskitoOptions {
```

Comment:
Do we really require getter/setter ? 🤔 

Could we just do the following ?

```ts
@Input()
maskito: MaskitoOptions | null = null;

accessor.writeValue = (value: unknown) => {
            original(maskitoTransform(String(value ?? ''), this.maskito ?? MASKITO_DEFAULT_OPTIONS));
        };
```

Reply:
Opps, my mistake, sorry.

We should keep the link to the same object even if input prop is changed.